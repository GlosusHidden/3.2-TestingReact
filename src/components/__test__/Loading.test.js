import React from 'react'
import renderer from 'react-test-renderer'

import Loading from '../Loading'

it('Loading Component Snapshot', () => {
  const renderedValue = renderer.create(
    <Loading />
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
