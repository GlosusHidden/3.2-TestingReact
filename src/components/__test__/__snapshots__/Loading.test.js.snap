// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component Snapshot 1`] = `
<div
  className="placeholder"
>
  <div
    className="text"
  >
    Wait, weather loading is in progress..
  </div>
  <img
    alt="loading"
    className="placeholder_img"
    src="img/loading.gif"
  />
</div>
`;
